packages/web-runtime/themes/owncloud/theme.json

```diff
@@ -100,7 +100,7 @@
       "appBanner": {
         "title": "ownCloud",
         "publisher": "ownCloud GmbH",
-        "price": "FREE",
+        "additionalInformation": "FREE",
```

Comment:
But FREE ls still untranslated like this...

Reply:
umm, not really. it is wrapped in gettext so it can be translated. how would you include all translations in theme.json?

Reply:
It's not wrapped, if this gonna be wrapped later via a gettext(themeconf.additional). It will not land into transifex. Because a dynamic gettext extract doesn't exist.

Reply:
as a matter of a fact: it is. please check the code. it will not go into transifex but this is expected behavior since we can't translate every imaginable entry a client will configure.

Reply:
What ? 😅